Index: code_samples/conditional_operator_intention/build.gradle
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- code_samples/conditional_operator_intention/build.gradle	(revision 1f48aabfc42ea24cd75520b6c2e1fe0315b33db9)
+++ code_samples/conditional_operator_intention/build.gradle	(date 1586234381935)
@@ -26,7 +26,7 @@
 
 // See https://github.com/JetBrains/gradle-intellij-plugin/
 intellij {
-  version '2019.3.4'
+  version '2020.1'
   plugins 'java'
   sameSinceUntilBuild = true
 }
Index: code_samples/psi_demo/build.gradle
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- code_samples/psi_demo/build.gradle	(revision 1f48aabfc42ea24cd75520b6c2e1fe0315b33db9)
+++ code_samples/psi_demo/build.gradle	(date 1586234381926)
@@ -16,7 +16,7 @@
 
 // See https://github.com/JetBrains/gradle-intellij-plugin/
 intellij {
-  version '2019.3.4'
+  version '2020.1'
   plugins 'java'
   sameSinceUntilBuild = true
 }
Index: code_samples/module/build.gradle
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- code_samples/module/build.gradle	(revision 1f48aabfc42ea24cd75520b6c2e1fe0315b33db9)
+++ code_samples/module/build.gradle	(date 1586234381945)
@@ -16,7 +16,7 @@
 
 // See https://github.com/JetBrains/gradle-intellij-plugin/
 intellij {
-  version '2019.3.4'
+  version '2020.1'
   sameSinceUntilBuild = true
 }
 
Index: code_samples/action_basics/build.gradle
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- code_samples/action_basics/build.gradle	(revision 1f48aabfc42ea24cd75520b6c2e1fe0315b33db9)
+++ code_samples/action_basics/build.gradle	(date 1586234381916)
@@ -16,7 +16,7 @@
 
 // See https://github.com/JetBrains/gradle-intellij-plugin/
 intellij {
-  version '201-EAP-SNAPSHOT'
+  version '2020.1'
   sameSinceUntilBuild = true
 }
 patchPluginXml {
Index: code_samples/tool_window/build.gradle
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- code_samples/tool_window/build.gradle	(revision 1f48aabfc42ea24cd75520b6c2e1fe0315b33db9)
+++ code_samples/tool_window/build.gradle	(date 1586234381932)
@@ -16,7 +16,7 @@
 
 // See https://github.com/JetBrains/gradle-intellij-plugin/
 intellij {
-  version '2019.3.4'
+  version '2020.1'
   sameSinceUntilBuild = true
 }
 
Index: code_samples/comparing_references_inspection/build.gradle
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- code_samples/comparing_references_inspection/build.gradle	(revision 1f48aabfc42ea24cd75520b6c2e1fe0315b33db9)
+++ code_samples/comparing_references_inspection/build.gradle	(date 1586234381929)
@@ -26,7 +26,7 @@
 
 // See https://github.com/JetBrains/gradle-intellij-plugin/
 intellij {
-  version '2019.3.4'
+  version '2020.1'
   plugins 'java'
   sameSinceUntilBuild = true
 }
Index: code_samples/max_opened_projects/build.gradle
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- code_samples/max_opened_projects/build.gradle	(revision 1f48aabfc42ea24cd75520b6c2e1fe0315b33db9)
+++ code_samples/max_opened_projects/build.gradle	(date 1586234381905)
@@ -16,7 +16,7 @@
 
 // See https://github.com/JetBrains/gradle-intellij-plugin/
 intellij {
-  version '2019.3.4'
+  version '2020.1'
   sameSinceUntilBuild = true
 }
 patchPluginXml {
Index: code_samples/facet_basics/build.gradle
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- code_samples/facet_basics/build.gradle	(revision 1f48aabfc42ea24cd75520b6c2e1fe0315b33db9)
+++ code_samples/facet_basics/build.gradle	(date 1586234381958)
@@ -16,7 +16,7 @@
 
 // See https://github.com/JetBrains/gradle-intellij-plugin/
 intellij {
-  version '2019.3.4'
+  version '2020.1'
   sameSinceUntilBuild = true
 }
 patchPluginXml {
Index: code_samples/run_configuration/build.gradle
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- code_samples/run_configuration/build.gradle	(revision 1f48aabfc42ea24cd75520b6c2e1fe0315b33db9)
+++ code_samples/run_configuration/build.gradle	(date 1586234381920)
@@ -16,7 +16,7 @@
 
 // See https://github.com/JetBrains/gradle-intellij-plugin/
 intellij {
-  version '2019.3.4'
+  version '2020.1'
   sameSinceUntilBuild = true
 }
 
Index: code_samples/editor_basics/build.gradle
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- code_samples/editor_basics/build.gradle	(revision 1f48aabfc42ea24cd75520b6c2e1fe0315b33db9)
+++ code_samples/editor_basics/build.gradle	(date 1586234381942)
@@ -16,7 +16,7 @@
 
 // See https://github.com/JetBrains/gradle-intellij-plugin/
 intellij {
-  version '2019.3.4'
+  version '2020.1'
   sameSinceUntilBuild = true
 }
 patchPluginXml {
Index: code_samples/framework_basics/build.gradle
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- code_samples/framework_basics/build.gradle	(revision 1f48aabfc42ea24cd75520b6c2e1fe0315b33db9)
+++ code_samples/framework_basics/build.gradle	(date 1586234381964)
@@ -16,7 +16,7 @@
 
 // See https://github.com/JetBrains/gradle-intellij-plugin/
 intellij {
-  version '2019.3.4'
+  version '2020.1'
   plugins 'java'
   sameSinceUntilBuild = true
 }
Index: code_samples/inspection_basics/build.gradle
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- code_samples/inspection_basics/build.gradle	(revision 1f48aabfc42ea24cd75520b6c2e1fe0315b33db9)
+++ code_samples/inspection_basics/build.gradle	(date 1586234381937)
@@ -16,7 +16,7 @@
 
 // See https://github.com/JetBrains/gradle-intellij-plugin/
 intellij {
-  version '2019.3.4'
+  version '2020.1'
   sameSinceUntilBuild = true
 }
 
Index: code_samples/project_model/build.gradle
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- code_samples/project_model/build.gradle	(revision 1f48aabfc42ea24cd75520b6c2e1fe0315b33db9)
+++ code_samples/project_model/build.gradle	(date 1586234381940)
@@ -16,7 +16,7 @@
 
 // See https://github.com/JetBrains/gradle-intellij-plugin/
 intellij {
-  version '2019.3.4'
+  version '2020.1'
   plugins 'java'
   sameSinceUntilBuild = true
 }
Index: code_samples/kotlin_demo/build.gradle
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- code_samples/kotlin_demo/build.gradle	(revision 1f48aabfc42ea24cd75520b6c2e1fe0315b33db9)
+++ code_samples/kotlin_demo/build.gradle	(date 1586234381967)
@@ -25,7 +25,7 @@
 
 // See https://github.com/JetBrains/gradle-intellij-plugin/
 intellij {
-  version '2019.3.4'
+  version '2020.1'
   sameSinceUntilBuild = true
 }
 compileKotlin {
Index: code_samples/live_templates/build.gradle
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- code_samples/live_templates/build.gradle	(revision 1f48aabfc42ea24cd75520b6c2e1fe0315b33db9)
+++ code_samples/live_templates/build.gradle	(date 1586234381950)
@@ -16,7 +16,7 @@
 
 // See https://github.com/JetBrains/gradle-intellij-plugin/
 intellij {
-  version '201-EAP-SNAPSHOT'
+  version '2020.1'
   sameSinceUntilBuild = true
 }
 patchPluginXml {
Index: code_samples/project_wizard/build.gradle
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- code_samples/project_wizard/build.gradle	(revision 1f48aabfc42ea24cd75520b6c2e1fe0315b33db9)
+++ code_samples/project_wizard/build.gradle	(date 1586234381953)
@@ -16,7 +16,7 @@
 
 // See https://github.com/JetBrains/gradle-intellij-plugin/
 intellij {
-  version '2019.3.4'
+  version '2020.1'
   sameSinceUntilBuild = true
 }
 
Index: code_samples/project_view_pane/build.gradle
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- code_samples/project_view_pane/build.gradle	(revision 1f48aabfc42ea24cd75520b6c2e1fe0315b33db9)
+++ code_samples/project_view_pane/build.gradle	(date 1586234381955)
@@ -16,7 +16,7 @@
 
 // See https://github.com/JetBrains/gradle-intellij-plugin/
 intellij {
-  version '2019.3.4'
+  version '2020.1'
   sameSinceUntilBuild = true
 }
 
Index: code_samples/tree_structure_provider/build.gradle
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- code_samples/tree_structure_provider/build.gradle	(revision 1f48aabfc42ea24cd75520b6c2e1fe0315b33db9)
+++ code_samples/tree_structure_provider/build.gradle	(date 1586234381961)
@@ -16,7 +16,7 @@
 
 // See https://github.com/JetBrains/gradle-intellij-plugin/
 intellij {
-  version '2019.3.4'
+  version '2020.1'
   sameSinceUntilBuild = true
 }
 
Index: code_samples/simple_language_plugin/build.gradle
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- code_samples/simple_language_plugin/build.gradle	(revision 1f48aabfc42ea24cd75520b6c2e1fe0315b33db9)
+++ code_samples/simple_language_plugin/build.gradle	(date 1586234381923)
@@ -30,7 +30,7 @@
 
 // See https://github.com/JetBrains/gradle-intellij-plugin/
 intellij {
-  version '2019.3.4'
+  version '2020.1'
   plugins 'java'
   sameSinceUntilBuild = true
 }
